pixel-point/pixelpoint-website
# feat: replace covers and fix code snippets for Blog (dark theme)

**Describe what changes this pull request brings**
This pull request removes white flash in dark theme when the page is reloading.

**Steps to test**

1. Open the [page]()
2. Confirm that everything looks and works as expected

**Screenshots**
N/A
